spec/services/notifications_list_spec.rb

```diff
@@ -35,6 +35,26 @@
 
         expect(NotificationsList.for(application_choice, include_ratifying_provider: true, event: :offer_accepted).to_a).to match_array([training_provider_user, ratifying_provider_user])
       end
+
+      it 'returns a provider user who is a member of the training and ratifying providers without duplicates' do
+        ratifying_provider = create(:provider)
+        training_provider = create(:provider)
+        provider_user = create(:provider_user, send_notifications: true, providers: [training_provider, ratifying_provider])
+        application_choice = create(:application_choice, course_option: create(:course_option, course: create(:course, provider: training_provider, accredited_provider: ratifying_provider)))
+
+        create(
+          :provider_user_notification_preferences,
+          :all_off,
+          provider_user: create(:provider_user, send_notifications: false, providers: [training_provider, ratifying_provider]),
+        )
+        create(:provider_user_notification_preferences)
```

Comment:
I don't think we need to test her that a user without notification prefs doesn't get notified? These could be

```ruby
        create(:provider_user_notification_preferences, provider_user: provider_user)
```